packages/shorebird_code_push_protocol/lib/src/models/user.dart

```diff
@@ -32,4 +34,7 @@ class User {
 
   /// Whether the user is currently a paying customer.
   final bool hasActiveSubscription;
+
+  /// The user's current subscription, if any.
+  final Subscription? subscription;
```

Comment:
Also do we want to tightly couple `User` and `Subscription` so that querying for a user also queries for the user's subscription? What if the user has more than one subscription in the future?

Reply:
Same here, resolving